Lua_basic example doesn't flash in b-l072z-lrwan1

<!--
If your issue is a usage question, please submit it to the user mailing-list
users@riot-os.org or to the developer mailing-list devel@riot-os.org.
If your issue is related to security, please submit it to the security
mailing-list security@riot-os.org.
-->

#### Description
The `lua_basic` example fails to flash on `b-l072z-lrwan1` in current RIOT master. It seems there's a tiny overflow that's not being detected by the build system.

<!--
Example: Cannot build gnrc_networking application for samr21-xpro board.
-->

#### Steps to reproduce the issue
Flash `lua_basic` example in `b-l072z-lrwan1`.

#### Expected results
<!--
Example: The gnrc_networking application builds on samr21-xpro.
-->
The application is flashed without errors
#### Actual results
<!--
Please paste or specifically describe the actual output.
-->
```
make RIOT_CI_BUILD=1 CC_NOCOLOR=1 --no-print-directory -C /home/jialamos/Development/RIOT/examples/lua_basic flash-only test
make[1]: Nothing to be done for 'Makefile.include'.
make[1]: Nothing to be done for 'Makefile.include'.
/home/jialamos/Development/RIOT/dist/tools/openocd/openocd.sh flash
### Flashing Target ###
Open On-Chip Debugger 0.10.0+dev-00226-g1c2e3d41d (2018-01-02-18:28)
Licensed under GNU GPL v2
For bug reports, read
	http://openocd.org/doc/doxygen/bugs.html
WARNING: interface/stlink-v2-1.cfg is deprecated, please switch to interface/stlink.cfg
hla_swd
adapter speed: 300 kHz
adapter_nsrst_delay: 100
Info : The selected transport took over low-level target control. The results might differ compared to plain JTAG/SWD
none separate
srst_only separate srst_nogate srst_open_drain connect_assert_srst
Info : Unable to match requested speed 300 kHz, using 240 kHz
Info : Unable to match requested speed 300 kHz, using 240 kHz
Info : clock speed 240 kHz
Info : STLINK v2 JTAG v28 API v2 SWIM v18 VID 0x0483 PID 0x374B
Info : using stlink api v2
Info : Target voltage: 3.252174
Info : stm32l0.cpu: hardware has 4 breakpoints, 2 watchpoints
    TargetName         Type       Endian TapName            State       
--  ------------------ ---------- ------ ------------------ ------------
 0* stm32l0.cpu        hla_target little stm32l0.cpu        reset
Info : Unable to match requested speed 300 kHz, using 240 kHz
Info : Unable to match requested speed 300 kHz, using 240 kHz
adapter speed: 240 kHz
target halted due to debug-request, current mode: Thread 
xPSR: 0xf1000000 pc: 0x08000414 msp: 0x20000200
auto erase enabled
Info : Device: STM32L0xx (Cat.5)
Info : STM32L flash has dual banks. Bank (0) size is 96kb, base address is 0x8000000
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000000e msp: 0x20000200
Warn : no flash bank found for address 8018000
Warn : no flash bank found for address 8023918
wrote 98304 bytes from file /home/jialamos/Development/RIOT/examples/lua_basic/bin/b-l072z-lrwan1/lua_basic.elf in 22.288465s (4.307 KiB/s)
target halted due to breakpoint, current mode: Thread 
xPSR: 0x61000000 pc: 0x2000002e msp: 0x20000200
Error: checksum mismatch - attempting binary compare
diff 0 address 0x08018000. Was 0x00 instead of 0x09
diff 1 address 0x08018001. Was 0x00 instead of 0x4b
diff 2 address 0x08018002. Was 0x00 instead of 0xa6
diff 3 address 0x08018003. Was 0x00 instead of 0x81
diff 4 address 0x08018004. Was 0x00 instead of 0xa3
diff 5 address 0x08018005. Was 0x00 instead of 0x62
diff 6 address 0x08018006. Was 0x00 instead of 0x09
diff 7 address 0x08018007. Was 0x00 instead of 0x4b
diff 8 address 0x08018008. Was 0x00 instead of 0x24
diff 9 address 0x08018009. Was 0x00 instead of 0x62
diff 10 address 0x0801800a. Was 0x00 instead of 0xe3
diff 11 address 0x0801800b. Was 0x00 instead of 0x62
diff 12 address 0x0801800c. Was 0x00 instead of 0x08
diff 13 address 0x0801800d. Was 0x00 instead of 0x4b
diff 14 address 0x0801800e. Was 0x00 instead of 0x23
diff 15 address 0x0801800f. Was 0x00 instead of 0x63
diff 16 address 0x08018010. Was 0x00 instead of 0xf3
diff 17 address 0x08018011. Was 0x00 instead of 0x05
diff 18 address 0x08018012. Was 0x00 instead of 0xdb
diff 19 address 0x08018013. Was 0x00 instead of 0xd5
diff 20 address 0x08018014. Was 0x00 instead of 0x02
diff 21 address 0x08018015. Was 0x00 instead of 0x23
diff 22 address 0x08018017. Was 0x00 instead of 0x22
diff 23 address 0x08018018. Was 0x00 instead of 0x21
diff 24 address 0x0801801a. Was 0x00 instead of 0x28
diff 25 address 0x0801801d. Was 0x00 instead of 0xf0
diff 26 address 0x0801801e. Was 0x00 instead of 0x4a
diff 27 address 0x0801801f. Was 0x00 instead of 0xf9
diff 28 address 0x08018020. Was 0x00 instead of 0xd4
diff 29 address 0x08018021. Was 0x00 instead of 0xe7
diff 30 address 0x08018022. Was 0x00 instead of 0xc0
diff 31 address 0x08018023. Was 0x00 instead of 0x46
diff 32 address 0x08018024. Was 0x00 instead of 0xbd
diff 33 address 0x08018025. Was 0x00 instead of 0xa4
diff 34 address 0x08018026. Was 0x00 instead of 0x01
diff 35 address 0x08018027. Was 0x00 instead of 0x08
diff 36 address 0x08018028. Was 0x00 instead of 0xe9
diff 37 address 0x08018029. Was 0x00 instead of 0xa4
diff 38 address 0x0801802a. Was 0x00 instead of 0x01
diff 39 address 0x0801802b. Was 0x00 instead of 0x08
diff 40 address 0x0801802c. Was 0x00 instead of 0x21
diff 41 address 0x0801802d. Was 0x00 instead of 0xa5
diff 42 address 0x0801802e. Was 0x00 instead of 0x01
diff 43 address 0x0801802f. Was 0x00 instead of 0x08
diff 44 address 0x08018030. Was 0x00 instead of 0x4d
diff 45 address 0x08018031. Was 0x00 instead of 0xa5
diff 46 address 0x08018032. Was 0x00 instead of 0x01
diff 47 address 0x08018033. Was 0x00 instead of 0x08
diff 48 address 0x08018034. Was 0x00 instead of 0x10
diff 49 address 0x08018035. Was 0x00 instead of 0xb5
diff 50 address 0x08018036. Was 0x00 instead of 0x03
diff 51 address 0x08018037. Was 0x00 instead of 0x4b
diff 52 address 0x08018038. Was 0x00 instead of 0x0a
diff 53 address 0x0801803a. Was 0x00 instead of 0x01
diff 54 address 0x0801803c. Was 0x00 instead of 0x18
diff 55 address 0x0801803d. Was 0x00 instead of 0x68
diff 56 address 0x0801803e. Was 0x00 instead of 0xff
diff 57 address 0x0801803f. Was 0x00 instead of 0xf7
diff 58 address 0x08018040. Was 0x00 instead of 0xbb
diff 59 address 0x08018041. Was 0x00 instead of 0xff
diff 60 address 0x08018042. Was 0x00 instead of 0x10
diff 61 address 0x08018043. Was 0x00 instead of 0xbd
diff 62 address 0x08018044. Was 0x00 instead of 0x28
diff 63 address 0x08018045. Was 0x00 instead of 0x02
diff 64 address 0x08018047. Was 0x00 instead of 0x20
diff 65 address 0x08018048. Was 0x00 instead of 0x0e
diff 66 address 0x08018049. Was 0x00 instead of 0xb4
diff 67 address 0x0801804a. Was 0x00 instead of 0x03
diff 68 address 0x0801804b. Was 0x00 instead of 0xb5
diff 69 address 0x0801804c. Was 0x00 instead of 0x01
diff 70 address 0x0801804e. Was 0x00 instead of 0x03
diff 71 address 0x0801804f. Was 0x00 instead of 0xab
diff 72 address 0x08018050. Was 0x00 instead of 0x04
diff 73 address 0x08018051. Was 0x00 instead of 0x48
diff 74 address 0x08018052. Was 0x00 instead of 0x04
diff 75 address 0x08018053. Was 0x00 instead of 0xcb
diff 76 address 0x08018055. Was 0x00 instead of 0x68
diff 77 address 0x08018056. Was 0x00 instead of 0x01
diff 78 address 0x08018057. Was 0x00 instead of 0x93
diff 79 address 0x08018058. Was 0x00 instead of 0x01
diff 80 address 0x08018059. Was 0x00 instead of 0xf0
diff 81 address 0x0801805a. Was 0x00 instead of 0xbc
diff 82 address 0x0801805b. Was 0x00 instead of 0xf9
diff 83 address 0x0801805c. Was 0x00 instead of 0x02
diff 84 address 0x0801805d. Was 0x00 instead of 0xb0
diff 85 address 0x0801805e. Was 0x00 instead of 0x08
diff 86 address 0x0801805f. Was 0x00 instead of 0xbc
diff 87 address 0x08018060. Was 0x00 instead of 0x03
diff 88 address 0x08018061. Was 0x00 instead of 0xb0
diff 89 address 0x08018062. Was 0x00 instead of 0x18
diff 90 address 0x08018063. Was 0x00 instead of 0x47
diff 91 address 0x08018064. Was 0x00 instead of 0x28
diff 92 address 0x08018065. Was 0x00 instead of 0x02
diff 93 address 0x08018067. Was 0x00 instead of 0x20
diff 94 address 0x08018068. Was 0x00 instead of 0xf0
diff 95 address 0x08018069. Was 0x00 instead of 0xb5
diff 96 address 0x0801806a. Was 0x00 instead of 0x15
diff 97 address 0x0801806c. Was 0x00 instead of 0x5d
diff 98 address 0x0801806d. Was 0x00 instead of 0x43
diff 99 address 0x0801806e. Was 0x00 instead of 0x85
diff 100 address 0x0801806f. Was 0x00 instead of 0xb0
diff 101 address 0x08018070. Was 0x00 instead of 0x07
diff 102 address 0x08018072. Was 0x00 instead of 0x01
diff 103 address 0x08018073. Was 0x00 instead of 0x91
diff 104 address 0x08018074. Was 0x00 instead of 0x02
diff 105 address 0x08018075. Was 0x00 instead of 0x92
diff 106 address 0x08018076. Was 0x00 instead of 0x03
diff 107 address 0x08018077. Was 0x00 instead of 0x93
diff 108 address 0x08018078. Was 0x00 instead of 0x0a
diff 109 address 0x08018079. Was 0x00 instead of 0x9c
diff 110 address 0x0801807b. Was 0x00 instead of 0x2d
diff 111 address 0x0801807c. Was 0x00 instead of 0x43
diff 112 address 0x0801807d. Was 0x00 instead of 0xd0
diff 113 address 0x0801807f. Was 0x00 instead of 0x28
diff 114 address 0x08018080. Was 0x00 instead of 0x04
diff 115 address 0x08018081. Was 0x00 instead of 0xd0
diff 116 address 0x08018082. Was 0x00 instead of 0x83
diff 117 address 0x08018083. Was 0x00 instead of 0x69
diff 118 address 0x08018085. Was 0x00 instead of 0x2b
diff 119 address 0x08018086. Was 0x00 instead of 0x01
diff 120 address 0x08018087. Was 0x00 instead of 0xd1
diff 121 address 0x08018088. Was 0x00 instead of 0xff
diff 122 address 0x08018089. Was 0x00 instead of 0xf7
diff 123 address 0x0801808a. Was 0x00 instead of 0x22
diff 124 address 0x0801808b. Was 0x00 instead of 0xff
diff 125 address 0x0801808c. Was 0x00 instead of 0x1f
diff 126 address 0x0801808d. Was 0x00 instead of 0x4b
diff 127 address 0x0801808e. Was 0x00 instead of 0x9c
More than 128 errors, the rest are not printed.

make: *** [/home/jialamos/Development/RIOT/examples/lua_basic/../../Makefile.include:513: flash] Error 1

Return value: 2
```
#### Versions
<!--
Operating system: Mac OSX, Linux, Vagrant VM
Build environment: GCC, CLang versions (you can run the following command from
the RIOT base directory: ./dist/tools/ci/print_toolchain_versions.sh).
-->
```
Operating System Environment
-----------------------------
       Operating System: "Arch Linux" 
                 Kernel: Linux 4.18.5-arch1-1-ARCH x86_64 unknown

Installed compiler toolchains
-----------------------------
             native gcc: gcc (GCC) 7.2.1 20171128
      arm-none-eabi-gcc: arm-none-eabi-gcc (GNU Tools for Arm Embedded Processors 7-2017-q4-major) 7.2.1 20170904 (release) [ARM/embedded-7-branch revision 255204]
                avr-gcc: missing
       mips-mti-elf-gcc: missing
             msp430-gcc: missing
   riscv-none-embed-gcc: missing
                  clang: clang version 7.0.0 (tags/RELEASE_700/final)

Installed compiler libs
-----------------------
   arm-none-eabi-newlib: "2.5.0"
    mips-mti-elf-newlib: missing
riscv-none-embed-newlib: missing
               avr-libc: missing (missing)

Installed development tools
---------------------------
                  cmake: cmake version 3.12.1
               cppcheck: Cppcheck 1.84
                doxygen: 1.8.14
                 flake8: missing
                    git: git version 2.18.0
                openocd: Open On-Chip Debugger 0.10.0+dev-00226-g1c2e3d41d (2018-01-02-18:28)
                 python: Python 3.7.0
                python2: Python 2.7.15
                python3: Python 3.7.0
             coccinelle: missing
```
<!-- Thanks for contributing! -->

--- a/boards/b-l072z-lrwan1/Makefile.features
+++ b/boards/b-l072z-lrwan1/Makefile.features
@@ -6,6 +6,12 @@ FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
 
+# Put other features for this board (in alphabetical order)
+# For riotboot you need an openocd that supports dualbank flashing.
+# The 0.10.0 openocd version in Ubuntu Bionic doesn't work. The change was
+# introduced after Jun 8, 2017 - v0.10.0-1-20170607-2132-dev.
+FEATURES_PROVIDED += riotboot
+
 # The board MPU family (used for grouping by the CI system)
 FEATURES_MCU_GROUP = cortex_m0_1
 

--- a/boards/nucleo-l073rz/Makefile.features
+++ b/boards/nucleo-l073rz/Makefile.features
@@ -9,8 +9,9 @@ FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
 
 # Put other features for this board (in alphabetical order)
-# For riotboot you need an openocd that supports dualbank flashing
-# (version >= v0.10.0)
+# For riotboot you need an openocd that supports dualbank flashing.
+# The 0.10.0 openocd version in Ubuntu Bionic doesn't work. The change was
+# introduced after Jun 8, 2017 - v0.10.0-1-20170607-2132-dev.
 FEATURES_PROVIDED += riotboot
 
 # load the common Makefile.features for Nucleo boards

--- a/boards/nucleo-l073rz/Makefile.features
+++ b/boards/nucleo-l073rz/Makefile.features
@@ -8,6 +8,11 @@ FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
 
+# Put other features for this board (in alphabetical order)
+# For riotboot you need an openocd that supports dualbank flashing
+# (version >= v0.10.0)
+FEATURES_PROVIDED += riotboot
+
 # load the common Makefile.features for Nucleo boards
 include $(RIOTBOARD)/common/nucleo64/Makefile.features
 

--- a/cpu/stm32l0/include/cpu_conf.h
+++ b/cpu/stm32l0/include/cpu_conf.h
@@ -41,6 +41,7 @@ extern "C" {
 #else
 #define CPU_IRQ_NUMOF                   (32U)
 #endif
+#define CPU_FLASH_BASE                  FLASH_BASE
 /** @} */
 
 /**

